modules/asset_import/assimp/config.h

```diff
@@ -0,0 +1,980 @@
+/*
+---------------------------------------------------------------------------
+Open Asset Import Library (assimp)
```

Comment:
Ah, so this is the `config.h` that should go in `thirdparty/assimp` in place of `config.h.in`.

Reply:
Moved to thirdparty/assimp/assimp/config.h